qiskit/providers/fake_provider/fake_backend.py

```diff
@@ -119,10 +119,15 @@ def _parse_channels(self, channels):
     def _setup_sim(self):
         if _optionals.HAS_AER:
```

Comment:
Only aer > 0.11.0 supports BackendV2 so you may need to check the version of Aer.

Reply:
Thanks but I'm not sure the updated code works well with aer < 0.11.0. Have you tested it with aer < 0.11.0? I guess the same version check may be necessary in `_setup_sim`.
@HuangJunye With this PR, we are changing to require aer >= 0.11.2 for FackBackendV2 users who want to do noisy simulation. I personally don't think it's too restrictive so I'm fine with it but I'd like to hear what others think about it.

Reply:
This is a new feature and Aer is just an optional accelerator/extender here, so it will be ok if it _can't_ be made to work with Aer < 0.11 - this PR won't go out until at least Terra 0.23 anyway.  However, it might not hurt to leave in the old function for a release, and continue to use it if Aer 0.10 (or whatever the minimum supported version is) is present.

With the inline version parsing here, I'm considering adding versioning methods to the lazy import checkers, which should simplify some of this.  I've made #9086 to track that.

Reply:
Yes, they would be a separate ticket (I made #9086 to track it, and I'll add the feature when I have time).  Terra 0.23.0 hasn't been released yet - it's due in January.  We released Terra 0.22.3 yesterday, which might have been the confusion.